@sanposhiho Thank you for reviewing. I modified it to use cmd.Diff on L657-L659.
example output is as follows:

```
make test WHAT=./pkg/scheduler/framework/plugins/volumebinding | grep -v "Mutation detector is enabled, this will result in memory leakage."
+++ [0513 16:44:16] Building go targets for linux/amd64
    k8s.io/kubernetes/hack/make-rules/helpers/go2make (non-static)
+++ [0513 16:45:04] Running tests without code coverage and with -race
--- FAIL: TestVolumeBinding (0.21s)
    --- FAIL: TestVolumeBinding/pod_has_not_pvcs (0.11s)
        volume_binding_test.go:623: Feed testing data and wait for them to be synced
        volume_binding_test.go:637: Start informer factory after initialization
        volume_binding_test.go:640: Wait for all started informers' cache were synced
        volume_binding_test.go:643: Verify
        volume_binding_test.go:654: Verify: call PreFilter and check status
        volume_binding_test.go:657: filter prefilter status does not match: (-want,+got):
              (*framework.Status)(
            -   &{
            -           code:    s"UnschedulableAndUnresolvable",
            -           reasons: []string{`persistentvolumeclaim "pvc-a" not found`},
            -   },
            +   nil,
              )
        volume_binding_test.go:664: Verify: check state after prefilter phase
        volume_binding_test.go:677: Verify: call Filter and check status
        volume_binding_test.go:685: Verify: Score
    --- FAIL: TestVolumeBinding/PVC_does_not_exist (0.11s)
        volume_binding_test.go:623: Feed testing data and wait for them to be synced
        volume_binding_test.go:637: Start informer factory after initialization
        volume_binding_test.go:640: Wait for all started informers' cache were synced
        volume_binding_test.go:643: Verify
        volume_binding_test.go:654: Verify: call PreFilter and check status
        volume_binding_test.go:657: filter prefilter status does not match: (-want,+got):
              (*framework.Status)(
            -   &{
            -           code:    s"UnschedulableAndUnresolvable",
            -           reasons: []string{`persistentvolumeclaim "pvc-a" not found - test`},
            -   },
            +   &{
            +           code:    s"UnschedulableAndUnresolvable",
            +           reasons: []string{`persistentvolumeclaim "pvc-a" not found`},
            +   },
              )
FAIL
FAIL    k8s.io/kubernetes/pkg/scheduler/framework/plugins/volumebinding 19.552s
FAIL
```

--- a/pkg/scheduler/framework/plugins/volumebinding/volume_binding_test.go
+++ b/pkg/scheduler/framework/plugins/volumebinding/volume_binding_test.go
@@ -18,7 +18,6 @@ package volumebinding
 
 import (
 	"context"
-	"reflect"
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
@@ -654,8 +653,8 @@ func TestVolumeBinding(t *testing.T) {
 
 			t.Logf("Verify: call PreFilter and check status")
 			_, gotPreFilterStatus := p.PreFilter(ctx, state, item.pod)
-			if !reflect.DeepEqual(gotPreFilterStatus, item.wantPreFilterStatus) {
-				t.Errorf("filter prefilter status does not match: %v, want: %v", gotPreFilterStatus, item.wantPreFilterStatus)
+			if diff := cmp.Diff(item.wantPreFilterStatus, gotPreFilterStatus); diff != "" {
+				t.Errorf("prefilter status does not match: (-want,+got):\n%s", diff)
 			}
 			if !gotPreFilterStatus.IsSuccess() {
 				// scheduler framework will skip Filter if PreFilter fails
@@ -678,8 +677,8 @@ func TestVolumeBinding(t *testing.T) {
 			t.Logf("Verify: call Filter and check status")
 			for i, nodeInfo := range nodeInfos {
 				gotStatus := p.Filter(ctx, state, item.pod, nodeInfo)
-				if !reflect.DeepEqual(gotStatus, item.wantFilterStatus[i]) {
-					t.Errorf("filter status does not match for node %q, got: %v, want: %v", nodeInfo.Node().Name, gotStatus, item.wantFilterStatus)
+				if diff := cmp.Diff(item.wantFilterStatus[i], gotStatus); diff != "" {
+					t.Errorf("filter status does not match for node %q, (-want,+got):\n%s", nodeInfo.Node().Name, diff)
 				}
 			}
 

